### PR TITLE
Vulnerability package fixes in various environments

### PR DESCRIPTION
**Pull request by skasturi**
Fixing vulnerabilities

### CODE REVIEW

**Review by ShipraJain01 — APPROVED** [review]

**Review by jpmann — DISMISSED** [review]

File: assets/training/finetune_acft_hf_nlp/environments/acpt/context/requirements.txt [hunk 1]
```diff
@@ -14,3 +14,9 @@ rouge-score==0.1.2
 sacrebleu==2.2.1
 bitsandbytes==0.38.1
 einops==0.6.1
+cryptography==41.0.2
+scipy==1.10.0
+aiohttp===3.8.5
+certifi===2023.07.22
+torch==1.13.1
```
Comment (amangupta26) [inline assets/training/finetune_acft_hf_nlp/environments/acpt/context/requirements.txt:21]: Do we need these? Torch is already baked as part of acpt base image.

And we have following three afected packages: cryptography, scipy, numpy

File: assets/training/model_evaluation/environment/context/conda.yaml [hunk 2]
```diff
@@ -25,4 +25,8 @@ dependencies:
   - statsmodels==0.11.0
   - lightgbm==3.2.1
   - msrest==0.6.21
-  - mltable=={{latest-pypi-version}}
\ No newline at end of file
+  - mltable=={{latest-pypi-version}}
+  - cryptography==41.0.2
+  - scipy==1.10.0
```
Comment (ShipraJain01) [inline assets/training/model_evaluation/environment/context/conda.yaml:30]: Please remove this, we dont need to add it in ME component

File: assets/training/model_evaluation/environment/context/conda.yaml [hunk 3]
```diff
@@ -25,4 +25,8 @@ dependencies:
   - statsmodels==0.11.0
   - lightgbm==3.2.1
   - msrest==0.6.21
-  - mltable=={{latest-pypi-version}}
\ No newline at end of file
+  - mltable=={{latest-pypi-version}}
+  - cryptography==41.0.2
```
Comment (ShipraJain01) [inline assets/training/model_evaluation/environment/context/conda.yaml:29]: why are we adding this explicitly? this was never a dependency in ME components. @skasturi

File: assets/training/model_evaluation/environment/context/conda.yaml [hunk 4]
```diff
@@ -25,4 +25,8 @@ dependencies:
...
+  - scipy==1.10.0
+  - aiohttp===3.8.5
```
Comment (ShipraJain01) [inline assets/training/model_evaluation/environment/context/conda.yaml:31]: why are we adding this explicitly? this was never a dependency in ME components. @skasturi

File: assets/training/model_evaluation/environment/context/conda.yaml [hunk 5]
```diff
@@ -25,4 +25,8 @@ dependencies:
...
+  - certifi===2023.07.22
```
Comment (ShipraJain01) [inline assets/training/model_evaluation/environment/context/conda.yaml:32]: why are we adding this explicitly? this was never a dependency in ME components. @skasturi .. Is this a new requirement from azureml-assets?

**Review by ShipraJain01 — CHANGES_REQUESTED** [review]

File: assets/training/finetune_acft_hf_nlp/environments/acpt/context/requirements.txt (hunk 1 above)
Comment (skasturi) [reply assets/training/finetune_acft_hf_nlp/environments/acpt/context/requirements.txt:21]: Shared on Teams. All the above pinned packages are affected.

**Review by amangupta26 — APPROVED** [review]

File: assets/training/model_evaluation/environment/context/conda.yaml (hunk 2 above)
Comment (skasturi) [reply assets/training/model_evaluation/environment/context/conda.yaml]: Done

File: assets/training/model_evaluation/environment/context/conda.yaml (hunk 3 above)
Comment (skasturi) [reply assets/training/model_evaluation/environment/context/conda.yaml]: Done. Reverted model evaluation.

File: assets/training/model_evaluation/environment/context/conda.yaml (hunk 4 above)
Comment (skasturi) [reply assets/training/model_evaluation/environment/context/conda.yaml]: Done. Reverted model evaluation.

File: assets/training/model_evaluation/environment/context/conda.yaml (hunk 5 above)
Comment (skasturi) [reply assets/training/model_evaluation/environment/context/conda.yaml]: Done. Reverted model evaluation.

**Review by vizhur — APPROVED** [review]

**Pull request merged**